lib/config/types.ts

```diff
@@ -80,13 +80,19 @@ export interface GlobalOnlyConfig {
   repositories?: RenovateRepository[];
 }
 
+export enum BinarySource {
+  Docker = 'docker',
+  Global = 'global',
+}
```

Comment:
Can we use a type def instead, or move this to `lib/types/...` otherwise we cant use type imports 😕